run/pubsub/README.md

```diff
@@ -9,4 +9,6 @@ For more details on how to work with this sample read the [Google Cloud Run Node
 ## Dependencies
 
 * **express**: Web server framework.
```

Comment:
I'm wondering if this list is useful or just a hassle to keep up to date. We might consider deleting it in the future.

Reply:
I would remove this section as we aren't consistently using it throughout this repo. Instead, we can comment in the files where they are used.

Reply:
It's consistent in the Cloud Run samples. While the really common dependencies like express may be fairly obvious, I'd argue that it is valuable to facilitate maintenance by explaining why dependencies are present, and it may be worth the hassle to expand the practice. Let's make Cloud Run a test?